src/base/ops.rs

```diff
@@ -374,6 +374,46 @@ componentwise_binop_impl!(Sub, sub, ClosedSub;
                           SubAssign, sub_assign, sub_assign_statically_unchecked, sub_assign_statically_unchecked_mut;
                           sub_to, sub_to_statically_unchecked);
 
+/// Matrices and vectors implement the `iter::Sum` trait. 
+/// 
+/// # Static matrices
+///
+/// ```.rust
+///use nalgebra::Matrix3x2;
```

Comment:
In nalgebra's doctests, it's been customary to hide the imports from the rendered code snippets; it helps reduce visual noise when they're rendered as html. To do this, change this import to:
```
#use nalgebra::Matrix3x2;
```